README.md

```diff
@@ -52,6 +52,73 @@ following fields:
 * `events`: An event sink which supports the same interface as Node's `EventEmitter`. Required. See
   below for the events that get generated.
 
+### Emitting metadata about the document
+
+Every time the validator finds/deduces a piece of metadata about the document, it emits a `metadata` event.
+`metadata` messages contain two arguments: *key* and *value*.
+Keys are unique ID's, while the types of values are different according to the specific kind of metadata.
+
+These properties are now returned when found:
+
+* `docDate`: The date associated to the document.
+* `title`: The (possible) title of the document.
+* `process`: The process rules, **as they appear on the text of the document**, eg `'14 October 2005'`.
+* `group`: The group(s) reponsible for the document (*deliverers*).
+* `thisVersion`: URL for this version of the document.
+* `previousVersion`: URL for the immediately previous version of the document.
+* `latestVersion`: URL for the latest version of the document.
+* `editorIDs`: ID(s) of the editor(s) responsible for the document.
+
+As an example, validating [`http://www.w3.org/TR/2014/REC-exi-profile-20140909/`](http://www.w3.org/TR/2014/REC-exi-profile-20140909/) (REC)
+emits these pairs of metadata:
+
+```javascript
+{ docDate: Tue Sep 09 2014 00:00:00 GMT+0900 (JST) }
+{ title: 'Efficient XML Interchange (EXI) Profile for limiting usage of dynamic memory' }
+{ thisVersion: 'http://www.w3.org/TR/2014/REC-exi-profile-20140909/' }
+{ latestVersion: 'http://www.w3.org/TR/exi-profile/' }
+{ previousVersion: 'http://www.w3.org/TR/2014/PR-exi-profile-20140506/' }
+{ editorIDs: [] }
+{ process: '14 October 2005' }
+{ group: { 'http://www.w3.org/XML/EXI/': 'Efficient XML Interchange Working Group' } }
+```
+
+If you download that very spec, edit it to include the following metadata&hellip;
+
+```html
+<dt>Editors:</dt>
+<dd data-editor-id="329883">Youenn Fablet, Canon Research Centre France</dd>
+<dd data-editor-id="foo bar baz">Daniel Peintner, Siemens AG</dd>
+```
+
+&hellip;and serve it locally from your machine, *Specberus* will spit also editor IDs:
+
+```javascript
+{ docDate: Tue Sep 09 2014 00:00:00 GMT+0900 (JST) }
+{ title: 'Efficient XML Interchange (EXI) Profile for limiting usage of dynamic memory' }
+{ latestVersion: 'http://www.w3.org/TR/exi-profile/' }
+{ previousVersion: 'http://www.w3.org/TR/2014/PR-exi-profile-20140506/' }
+{ editorIDs: [ '329883', 'foo bar baz' ] }
```

Comment:
I would say that if the ID is not a number, it is not valid. We could also argue that if it is bigger than a million, it is not valid either, that might reduce the number of typos until we have something better...